src/EditorFeatures/Core/Implementation/Suggestions/FixMultipleSuggestedAction.cs

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.Internal.Log;
+using Microsoft.VisualStudio.Text;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.Implementation.Suggestions
+{
+    /// <summary>
+    /// Suggested action for fix multiple occurrences code fix.
+    /// </summary>
+    internal class FixMultipleSuggestedAction : FixAllSuggestedAction
```

Comment:
Editor team exposes ISuggestedAction, and [SuggestedAction](http://source.roslyn.io/#Microsoft.CodeAnalysis.EditorFeatures/Implementation/Suggestions/SuggestedAction.cs,72ee6a717bc09afb) is our internal implementation in EditorFeatures layer.

Error list one, i.e. this type, is based on ISuggestedAction, and hence gets all the goodness of exception handling + FixAll preview dialog etc.